From 8aecbcf11a402f4591d5a9fd27825e51e4f5b915 Mon Sep 17 00:00:00 2001
From: Muhammad Husaini Zulkifli <muhammad.husaini.zulkifli@intel.com>
Date: Fri, 28 May 2021 12:06:54 +0800
Subject: [PATCH 3/3] ethtool: Fix usage options naming

- Fix wrong usage option naming for add-frag-size.
- Remove unused preemptible-queues-mask usage options.

Signed-off-by: Muhammad Husaini Zulkifli <muhammad.husaini.zulkifli@intel.com>
---
 ethtool.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/ethtool.c b/ethtool.c
index f08005a..2e75fa6 100644
--- a/ethtool.c
+++ b/ethtool.c
@@ -6057,8 +6057,7 @@ static const struct option args[] = {
 		.func	= do_set_preempt,
 		.help	= "Set Frame Preemption settings",
 		.xhelp	= "		[ fp on|off ]\n"
-			  "		[ preemptible-queues-mask %x ]\n"
-			  "		[ min-frag-size %d ]\n",
+			  "		[ add-frag-size %d ]\n",
 	},
 	{
 		.opts	= "-Q|--per-queue",
-- 
2.17.1

